.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
```

Comment:
It also seems kind of weird for this to be here, but IDK where else it could go.

Reply:
Yes, I've been debating this _a lot_ as well. iohk-nix is the ground truth for our libraries. As this is basically just taking what's _in_ the repo, and publishing it on GH. I wouldn't know how else to do it, other than starting to work this in hydra pushing stuff to GH, which is going to be messy. And then hydra specific :-/